lib/tasks/migrations/migrations.rake

```diff
@@ -74,6 +74,43 @@ namespace :migrations do
     exit has_encountred_issue
   end
 
+  task :rooms, [:start, :stop] => :environment do |_task, args|
+    start, stop = range(args)
+    has_encountred_issue = 0
+
+    Room.select(:id, :uid, :name, :bbb_id, :last_session, :user_id)
```

Comment:
I forgot this,
We'd need to migrate the rooms for the migrated users.
This will migrate rooms for pending, denied and deleted users.

Reply:
Good call, I will try to fix this by the end of the day.

Reply:
Discussed this with @farhatahmad and we will migrate all the rooms.
In v3, the rooms from banned/pending users won't be created as the user will not be find - it will return a bad_request error.

Reply:
EDIT: Working on a solution to store the migrated Users in a variable. Coming soon

Reply:
Done - add migrated_users as :namespace instance variable.

Reply:
EDIT: Rake task not working like I expected, reverting last commit.

Reply:
I don't think that we're not synchronized on this,
The migration system in place will NOT migrate any pending, denied, deleted or super admin users.
A room cannot be migrated to an inexistent user on v3 otherwise will get a bad request.
There's no way to migrate all rooms for inexistent users and a pending, denied,... users will be filtered out on users migration task.
For me all I see is that we either migrate all users including those inactive which is not aligned with what I discussed with @farhatahmad .
Or, we'll be having false negatives that will not resolve on reruns and a waste of performance due to errors caused by migrating rooms for inexistent users that will never exist and that will also ruin the migration auto repair property (rerunning the migration task will fix any failed individual migrations).

Reply:
I don't think that we're synchronized on this,
The migration system in place will NOT migrate any pending, denied, deleted or super admin users.
A room cannot be migrated to an inexistent user on v3 otherwise will get a bad request.
There's no way to migrate all rooms for inexistent users and a pending, denied,... users will be filtered out on users migration task.
For me all I see is that we either migrate all users including those inactive which is not aligned with what I discussed with @farhatahmad .
Or, we'll be having false negatives that will not resolve on reruns and a waste of performance due to errors caused by migrating rooms for inexistent users that will never exist and that will also ruin the migration auto repair property (rerunning the migration task will fix any failed individual migrations).

Reply:
```
role_ids = RESERVED_ROLE_NAMES.ids

Room.includes(:user)
.select(:id, :uid, :name, :bbb_id, :last_session, :user_id)
.where.not(user: { role_id: role_ids} )
.find_each(start: start, finish: stop, batch_size: COMMON[:batch_size]).each do |r|
```